src/factories/pair/uniswap-pair.factory.ts

```diff
@@ -333,14 +333,13 @@ export class UniswapPairFactory {
           ? null
           : bestRouteQuote.expectedConvertQuoteOrTokenAmountInMaxWithSlippage,
       expectedConvertQuote: bestRouteQuote.expectedConvertQuote,
-      liquidityProviderFee:
+      liquidityProviderFee: bestRouteQuote.liquidityProviderFee.map((f) =>
```

Comment:
breaking changes for v2.. if many liquidity provider fees the expected convert quote will be different depending on that no? would not expect a loop here i dont think...